doc/build/changelog/changelog_14.rst

```diff
@@ -1721,7 +1721,7 @@ This document details individual issue-level changes made throughout
         :paramref:`.FunctionElement.table_valued.joins_implicitly`, for the
         :meth:`.FunctionElement.table_valued` construct. This parameter indicates
         that the given table-valued function implicitly joins to the table it
-        refers towards, essentially disabling the "from linting" feature, i.e. the
+        refers to, essentially disabling the "from linting" feature, i.e. the
```

Comment:
so.....is this a dangling preposition?   "towards" is a prep also so that would be wrong as well.  so....sentences like this would need to be fixed